Microsoft.NET.Build.Containers/CreateNewImage.cs

```diff
@@ -109,6 +109,7 @@ public override bool Execute()
         
         Layer newLayer = Layer.FromDirectory(PublishDirectory, WorkingDirectory);
         image.AddLayer(newLayer);
+        image.WorkingDirectory = WorkingDirectory;
```

Comment:
😳

Reply:
yeah - I think for rc1 we should have some kind of output from this target (like the manifest or something) that we can programmatically inspect. That would make it easier to test this kind of invariant.